### PR TITLE
Update to Gradle 5.3.1

### PR DESCRIPTION
**Pull request by pvliss**
Release Notes: https://docs.gradle.org/5.3.1/release-notes.html

-   Please make sure the below checklist is followed for Pull Requests.

-   [ ] [Travis tests](https://travis-ci.org/jhipster/generator-jhipster/pull_requests) are green
-   [x] Tests are added where necessary
-   [x] Documentation is added/updated where necessary
-   [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

File: generators/server/templates/gradlew [hunk 1]
```diff
@@ -28,7 +28,7 @@ APP_NAME="Gradle"
 APP_BASE_NAME=`basename "$0"`
 
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS='"-Xmx64m"'
+DEFAULT_JVM_OPTS='"-Xmx64m" "-Xms64m"'
```
Comment (pmverma) [inline generators/server/templates/gradlew]: Cann't this work with `"-Xmx64m -Xms64m"`, without adding many quotes?

File: generators/server/templates/gradlew.bat [hunk 2]
```diff
@@ -14,7 +30,7 @@ set APP_BASE_NAME=%~n0
 set APP_HOME=%DIRNAME%
 
 @rem Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-set DEFAULT_JVM_OPTS="-Xmx64m"
+set DEFAULT_JVM_OPTS="-Xmx64m" "-Xms64m"
```
Comment (pmverma) [inline generators/server/templates/gradlew.bat]: Cann't this work with `"-Xmx64m -Xms64m"`, without adding many quotes?

File: generators/server/templates/gradlew (hunk 1 above)
Comment (atomfrede) [reply generators/server/templates/gradlew]: This file is part of the gradle distribution, so we don't have any influence on how they ship this file.

File: generators/server/templates/gradlew.bat (hunk 2 above)
Comment (atomfrede) [reply generators/server/templates/gradlew.bat]: See abobe

**Comment by pvliss** [comment]
@atomfrede is right. These files are generated by gradle and we use them as they are. Nevertheless I forgot to change the wrapper version within `build.gradle`. Fixing it now

**Pull request merged**